docs/user-guide/creating-application/app-details.md

```diff
@@ -44,6 +42,12 @@ You can also copy the Ingress Host URL from the **URLs** instead of searching in
 3. Click **URLs**.
 4. You can view or copy the **URL** of the Ingress Host.
 
+## Manage Traffic
```

Comment:
The heading could be deployment visibility & actions. Inside that we should discuss about
- App status
- Deployment status
- Rollout status (And rollout status has some actions like; swap traffic, manager traffic etc)

Reply:
Similarly the structuring needs to be done the same way